taxonomies/ingredients.txt

```diff
@@ -47779,6 +47850,10 @@ it:cumino macinato
 openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:cumin-moulu
 # 52 products in 3 languages @2018-10-07
 
+<en:seed
```

Comment:
we can remove this entry: we should now be recognizing "XYZ en polvo" for all ingredients.

You can see it here:
https://es.openfoodfacts.org/cgi/test_ingredients_analysis.pl?ingredients_text=comino+en+polvo&type=add&action=process&.submit=Valider